### ISSUE TITLE
Could not parse swift file

### ISSUE THREAD

**Issue by corinnekrych** [jpsim/SourceKitten#36]
The error:
```
Parsing SyncClient.swift (1/1)
sourcekit: [1:connection-event-handler:5907: 0.0000] Connection interrupt
sourcekit: [1:pingService:5907: 0.0008] pinging service
sourcekit: [1:sourcekitd_send_request_sync:779: 0.0021] request dropped while restoring service
Could not parse `SyncClient.swift`. Please open an issue at https://github.com/jpsim/SourceKitten/issues with the file contents.
```
when tempting to make jazzy documentation for  https://github.com/aerogear/aerogear-ios-sync-client/blob/master/AeroGearSyncClient/SyncClient.swift#L25

the line causing the issue is :
```
public class SyncClient<CS:ClientSynchronizer, D:DataStore where CS.T == D.T, CS.D == D.D, CS.P.E == CS.D> : WebSocketDelegate
```
if you remove the where clause like:
```
public class SyncClient<CS:ClientSynchronizer, D:DataStore where CS.T == D.T, CS.D == D.D, CS.P.E == CS.D> : WebSocketDelegate
```

 parsing is ok.

**Comment by jpsim** [comment]
Hi @corinnekrych, thanks for reporting this. It seems to be a problem with how we generate compiler arguments for SourceKit requests, which makes it crash spectacularly in this case.

**Comment by jpsim** [comment]
@corinnekrych I believe I've now fixed this problem in #40. Thanks for reporting and please let me know if you run into any issues whatsoever.
